### DISCUSSION TITLE
openDTU übergreifende Statistik?

### DISCUSSION THREAD

**Discussion by spcqike** [tbnobody/OpenDTU#1749] [General]
Hallo,

beim Ausprobieren von evcc (https://github.com/evcc-io/evcc) ist mir die Gesamtstatistik der Community ins Auge gesprungen.
![grafik](https://github.com/tbnobody/OpenDTU/assets/50098985/a4fb8b41-9ee5-4436-8951-916c65a0d4a8)

Irgendwie find ich die Vorstellung cool, dass es sowas ja auch bei openDTU geben könnte. im Sinne von Gesamtinverterleistung (maximal mögliche Leistung aller gekoppelten Wechselrichter), Gesamtproduktion (aktuelle Einspeisung/Erzeugung), Gesamtertrag (Heute und insgesamt?, in kWh / MWh) und gespartes CO2. Es wäre interessant zu sehen, wie viel die Community mit ihren kleinen und großen Anlagen als "virtuelles Kraftwerk" erreichen kann.

Selbstverständlich als opt-in, wie in evcc. 

Wäre sowas möglich? Was braucht es für solch eine Statistik? Wie seht ihr das? Nettes Feature oder eher nicht sinnvoll?

Grüße

**Comment by 3DJupp** [comment]
Meinst du die "Gesamtheit der OpenDTU Installationen"? Bestimmt ist sowas möglich, die Frage nur, wer die Plattform betreibt, da das auch gehostet werden muss.
Ich fänd das eine coole Sache.
Sind natürlich nur "Nice2Have" Features, aber so könnte man sicher Aussagen treffen, wie viel jede:r im Durchschnitt erzeugt etc. 
🌞 ⚡
